ipv6/address.go

```diff
@@ -0,0 +1,115 @@
+package ipv6
+
+import (
+	"fmt"
+	"net"
+)
+
+const (
+	// addressSize is the number of bits that an IPv6 address takes
+	addressSize int = 128
+)
+
+// Address represents an IPv6 address
+type Address struct {
+	ui uint128
+}
+
+// AddressFromUint64 returns the IPv6 address from its two 64 bit unsigned representation
+func AddressFromUint64(high uint64, low uint64) Address {
+	return Address{uint128{high, low}}
+}
+
+// AddressFromBytes returns the IPv6 address from bytes
+func AddressFromBytes(s []byte) (Address, error) {
```

Comment:
I like the AddressFromUint16 idea, i'll implement that. I still think it might be helpful to expose the AddressFromBytes as part of the API what do you think?

Reply:
Let's leave the bytes one out. I don't want the API surface to be larger than it needs to be. In fact, if you notice other parts of the API that are just redundant, please point them out. I've been trimming it down over the last few weeks.

Smaller API surfaces are more approachable, in general.